Bookish.Web/Views/Home/Loans.cshtml

```diff
@@ -34,6 +25,13 @@
                 <td>@bookLoan.Author</td>
                 <td>@bookLoan.Isbn</td>
                 <td>@bookLoan.DueDate.ToShortDateString()</td>
+                <td>
+                    <form asp-action="ReturnBook" asp-route-copyID="@bookLoan.CopyId">
```

Comment:
Not very important as I assume it works, but this should be `asp-route-copyId` I think to be consistent with your naming 🙂